src/pages/library/common.js

```diff
@@ -133,7 +133,6 @@ const FilterBar = ({ name, onClear, onFilterByLetter, onFilterBySearchText, onSo
     filterType === filterOptions.input && div({ style: { display: 'flex', alignItems: 'center', flexDirection: 'row' } }, [
       h(DelayedSearchInput, {
         style: { borderRadius: 25, borderColor: colors.dark(0.2), width: '100%', maxWidth: 575, height: '3rem', marginRight: 20 },
-        debounceMs: 25,
```

Comment:
What effect does removing this parameter have?

Reply:
Changes the debounce from 25 ms to the component default of 250 ms